app/streaming/video/ffmpeg-renderers/eglvid.cpp

```diff
@@ -0,0 +1,480 @@
+// vim: noai:ts=4:sw=4:softtabstop=4:expandtab
+#define GL_GLEXT_PROTOTYPES
+
+#include "eglvid.h"
+
+#include "streaming/session.h"
+#include "path.h"
+
+#include <QDir>
+
+#include <Limelight.h>
+#include <unistd.h>
+
+#include <SDL_egl.h>
+#include <SDL_opengl.h>
+#include <SDL_syswm.h>
+#include <SDL_opengles2.h>
+#include <SDL_opengles2_gl2ext.h>
+
+/* TODO:
+ *  - handle more pixel formats
+ *  - code refacto/cleanup
+ *  - handle software decoding
+ *  - handle window resize
```

Comment:
What currently happens if you resize?

Reply:
There are two problems:
* the viewport cannot be shrinked (on sway at least, ie: the content exceed the window boundary if resizing down)
* I don't handle any kind of ratio so any window that would'nt match the src ratio would probably be stretched in some form

Reply:
As a mater of fact, yes (on `sway`, I will test it on `KWin` X11+Wayland soon enough).

![sdl-resize2](https://user-images.githubusercontent.com/950419/81469201-07f09600-91e4-11ea-97c3-c723dbbea060.png)
![sdl-resize](https://user-images.githubusercontent.com/950419/81469202-08892c80-91e4-11ea-90fb-cbeb976d4961.png)

Reply:
* KWin(X11): resize works as expected
* KWin(Wayland): I just can't resize the window

Reply:
Ok, looks like a bug in SDL. Also not a regression going from SDLRenderer to EGLRenderer, so I'm okay with it.